plugin/plugin.go

```diff
@@ -152,29 +148,34 @@ func (p *CLIPlugin) GenerateEnvelope(ctx context.Context, req *proto.GenerateEnv
 // if ContractVersion is not set, it will be set by the function.
 func (p *CLIPlugin) VerifySignature(ctx context.Context, req *proto.VerifySignatureRequest) (*proto.VerifySignatureResponse, error) {
 	logger := log.GetLogger(ctx)
-	logger.Debugf("Plugin verify-signature request: {ContractVersion: %v, TrustPolicy: %v, PluginConfig: %v, Signature: {CriticalAttributes: %v, UnprocessedAttributes: %v}}", req.ContractVersion, req.TrustPolicy, req.PluginConfig, req.Signature.CriticalAttributes, req.Signature.UnprocessedAttributes)
 	var resp proto.VerifySignatureResponse
 	if req.ContractVersion == "" {
 		req.ContractVersion = proto.ContractVersion
 	}
+
 	err := run(ctx, p.name, p.path, req, &resp)
 	logger.Debugf("Plugin verify-signature response: %+v", resp)
 	return &resp, err
 }
 
 func run(ctx context.Context, pluginName string, pluginPath string, req proto.Request, resp interface{}) error {
+	logger := log.GetLogger(ctx)
+
 	// serialize request
 	data, err := json.Marshal(req)
 	if err != nil {
 		return fmt.Errorf("%s: failed to marshal request object: %w", pluginName, err)
 	}
 
+	logger.Debugf("Plugin %s request: %+v", req.Command(), string(data))
```

Comment:
Some requests contain binary data and we may don't want to log them.

Reply:
Why not? Our spec says binary data would be base64 encoded.